README.md is not included in tarball, so comment out long description
author: Kenneth Hoste (Ghent University)
--- deap-1.0.2/setup.py.orig	2015-05-13 15:58:21.188127136 +0200
+++ deap-1.0.2/setup.py	2015-05-13 15:58:30.288556411 +0200
@@ -19,7 +19,7 @@
 setup(name='deap',
       version=deap.__revision__,
       description='Distributed Evolutionary Algorithms in Python',
-      long_description=read_md('README.md'),
+      #long_description=read_md('README.md'),
       author='deap Development Team',
       author_email='deap-users@googlegroups.com',
       url='https://www.github.com/deap',
